frog/imports/client/Wiki/helpers.js

```diff
@@ -105,12 +105,35 @@ const listWikis = async () => {
   });
 };
 
+// Function to get the pages of all pages in a wiki, returns [Title, ID]
+const listPages = async wikiDoc => {
+  const list = await new Promise(resolve =>
+    wikiDoc.fetch(err => {
+      if (err) {
+        // eslint-disable-next-line no-console
+        console.log('Unable to fetch');
```

Comment:
Instead of logging, we should reject the promise and provide an error message

Reply:
Made the respective changes